css/_defaults.scss

```diff
@@ -54,7 +54,12 @@ th,
 td {
   padding: var(--space-medium);
   padding-left: 0;
+  padding-right: 0;
   vertical-align: top;
+
+  &:not(:last-of-type) {
```

Comment:
Make sure the last cell in a table row can have content to the edge of the page.